aws-postgresql.yml

```diff
@@ -61,6 +61,14 @@ provision:
       See the RDS User Guide for details:
       https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#gp3-storage.
     default: 3000
+  - field_name: use_managed_admin_password
+    type: boolean
+    default: false
+    details: Weather to use AWS managed credentials for admin credentials.
+  - field_name: password_rotate_after
```

Comment:
It might be worth renaming this to make it clear that it's the admin password that gets rotated because we might rotate binding passwords one day

Reply:
`rotate_admin_password_after`?